sw/device/lib/testing/usb_testutils_controlep.h

```diff
@@ -94,6 +94,17 @@ status_t usb_testutils_controlep_init(usb_testutils_controlep_ctx_t *ctctx,
                                       const uint8_t *test_dscr,
                                       size_t test_dscr_len);
 
+/**
+ * Wait until the device configuration has been set by the host
```

Comment:
```suggestion
 * Wait until the device configuration has been set by the host.
```
Also applies on the follow lines.

Reply:
Done. I was, again, mimicking the existing code. Over time it will all get changed, I guess!